ocaml/sdk-gen/go/gen_go_helper.ml

```diff
@@ -169,32 +195,230 @@ module Json = struct
     | _ ->
         [("event", `Null); ("session", `Null)]
 
+  let of_result obj msg =
+    match msg.msg_result with
+    | None ->
+        `Null
+    | Some (t, _d) ->
+        if obj.name = "event" && String.lowercase_ascii msg.msg_name = "from"
```

Comment:
Why a special handling on `event.from` is here?

Reply:
We replace `EventRecord` with `EventBatch`  in `event.from` since we has enclose below:
```go
type EventBatch struct {
	Token          string
	ValidRefCounts map[string]int
	Events         []EventRecord
}
```

Reply:
More background is needed from Sola.